sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -153,6 +158,9 @@ def __init__(self, packages, options, environment_version, pipeline_url):
     # User agent information.
     self.proto.userAgent = dataflow.Environment.UserAgentValue()
     self.local = 'localhost' in self.google_cloud_options.dataflow_endpoint
+    self._proto_pipeline = proto_pipeline
+    self._sdk_image_overrides = (
```

Comment:
How about making `dict()` default value in function definition, instead of `_sdk_image_overrides`?

Reply:
Seems like that results in a lint error (Dangerous default value dict() (builtins.dict) as argument).

Reply:
OK. We can leave it as is.